src/main/java/rsb/methods/Combat.java

```diff
@@ -267,14 +268,73 @@ public boolean isAttacking(final RSNPC npc) {
 		return interact != null && interact.equals(npc.getAccessor());
 	}
 
+	public boolean isHPZero(final RSNPC npc) {
+		return npc.getHPPercent() == 0;
+	}
+
+	public boolean isAlive(final RSNPC npc) {
+		return npc.getHPPercent() > 0 || (isHPZero(npc) && isFinishable(npc));
+	}
+
+	public boolean isDead(final RSNPC npc) {
+		return isHPZero(npc) && !isFinishable(npc);
+	}
+
+	public boolean isBelowHP(final RSNPC npc, final double healthPercentage) {
+		return npc.getHPPercent() < healthPercentage;
+	}
+
+	public boolean isBelowOrAtHP(final RSNPC npc, final double healthPercentage) {
+		return npc.getHPPercent() <= healthPercentage;
+	}
+
+	public boolean isAboveHP(final RSNPC npc, final double healthPercentage) {
+		return npc.getHPPercent() > healthPercentage;
+	}
+
+	public boolean isAboveOrAtHP(final RSNPC npc, final double healthPercentage) {
+		return npc.getHPPercent() >= healthPercentage;
+	}
+
+	public boolean isLowHealth(final RSNPC npc) {
+		return isAboveHP(npc, 0) && isBelowHP(npc, 15);
+	}
+
+	public boolean isFinishable(final RSNPC npc) {
+		String[] finishableMobs = {
+				"Rockslug", "Desert Lizard", "Small Lizard", "Lizard", "Mutated Zygomite", "Ancient Zygomite", "Gargoyle"
```

Comment:
Are you sure you don't want to enumerate these values since you use them twice? Especially in a switch-case.